src/results/absorption_chiller.jl

```diff
@@ -29,14 +29,20 @@
 # *********************************************************************************
 """
 `AbsorptionChiller` results keys:
-- `size_kw` Power capacity size of the absorption chiller system [kW]
-- `size_ton`
-- `year_one_to_tes_series_ton`
-- `year_one_thermal_consumption_series`
-- `year_one_thermal_consumption_kwh`
-- `year_one_thermal_production_tonhour`
-- `year_one_electric_consumption_series`
-- `year_one_electric_consumption_kwh`
+- `size_kw` # Optimal power capacity size of the absorption chiller system [kW]
+- `size_ton` 
+- `production_to_tes_series_ton` 
+- `production_to_load_series_ton` 
+- `thermal_consumption_series_mmbtu_per_hour`
+- `annual_thermal_consumption_mmbtu`
+- `annual_thermal_production_tonhour`
+- `electric_consumption_series_kw`
+- `annual_electric_consumption_kwh`
+
+!!! note "'Series' and 'Annual' energy outputs are average annual"
```

Comment:
@Bill-Becker Does this note seem okay to you? I added it to all of the output files in which it's relevant.

Reply:
There is no degradation or year-to-year change in absorption chiller thermal production (or electric consumption), so average annual would be the same as year one. The note seems to imply that there is year-to-year differences even though it doesn't explicitly say that.

Reply:
true! I wasn't differentiating between the thermal and electric techs. I removed this note from Absorption and ExistingChiller in https://github.com/NREL/REopt.jl/pull/160/commits/cc64cee3e1f3672b71813c58a67307e7bdaa4720